--- beryl-core-0.1.9999.1/libberyldecoration/Makefile.am.orig	2007-02-01 00:22:16.000000000 +0100
+++ beryl-core-0.1.9999.1/libberyldecoration/Makefile.am	2007-02-04 15:00:29.052985522 +0100
@@ -2,6 +2,7 @@
 
 libberyldecoration_la_SOURCES = main.c
 libberyldecoration_la_LDFLAGS = -no-undefined
+libberyldecoration_la_LIBADD = -lXrender -lX11 -lm
 
 INCLUDES = -I$(top_srcdir)/include
 
--- beryl-core-0.1.9999.1/libberylsettings/Makefile.am.orig	2007-02-01 00:22:15.000000000 +0100
+++ beryl-core-0.1.9999.1/libberylsettings/Makefile.am	2007-02-04 14:59:28.413529876 +0100
@@ -13,4 +13,4 @@
 
 lib_LTLIBRARIES=libberylsettings.la
 
-libberylsettings_la_LIBADD = @GLIB_LIBS@
+libberylsettings_la_LIBADD = @GLIB_LIBS@ -lX11 -ldl
diff -ruN beryl-core-0.1.4./berylsettings.pc.in beryl-core-0.1.4/berylsettings.pc.in
--- beryl-core-0.1.4./berylsettings.pc.in	2006-12-27 00:44:00.000000000 +0100
+++ beryl-core-0.1.4/berylsettings.pc.in	2006-12-27 01:13:16.797293649 +0100
@@ -7,6 +7,8 @@
 Description: OpenGL compositing manager
 Version: @VERSION@
 
-Requires: @BERYL_REQUIRES@ beryl glib-2.0
-Libs: @BERYL_LIBS@ @GLIB_LIBS@ -L@libdir@ -lberylsettings
+Requires: glib-2.0
+Requires.private: x11
+Libs: -L@libdir@ -lberylsettings
+Libs.private: -ldl
 Cflags: @BERYL_CFLAGS@ @GLIB_CFLAGS@ -I${includedir}/beryl
