--- beryl-core/src/main.c.orig	2006-09-29 12:50:48.000000000 +0200
+++ beryl-core/src/main.c	2006-09-30 22:25:50.000000000 +0200
@@ -75,8 +75,8 @@
 CompWindow *lastDamagedWindow = 0;
 
 Bool replaceCurrentWm = TRUE;
-Bool indirectRendering = FALSE;
-Bool strictBinding = FALSE;
+Bool indirectRendering = TRUE;
+Bool strictBinding = TRUE;
 Bool useCow = FALSE;
 Bool skipGlYield = FALSE;
 Bool isNVTFP = FALSE;
